python/mathsys/main/ir.py

```
@@ -30,39 +33,45 @@
-    return bytes([
-        value & 0xFF,
-        (value >> 8) & 0xFF,
-        (value >> 16) & 0xFF,
-        (value >> 24) & 0xFF
-    ])
-
-# NODES -> NAMESPACE
+# TYPES -> U8 CLASS
+class u8:
+    def __new__(self, value: int) -> bytes:
+        if not 1 <= value <= 2**8 - 1: raise ValueError(f"'{value}' is outside range for u8.")
```

Ruff (0.12.2)

Multiple statements on one line (colon)

(E701)